app/pom.xml

```diff
@@ -189,6 +193,16 @@
             <artifactId>lombok</artifactId>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>org.bouncycastle</groupId>
+            <artifactId>bcprov-jdk18on</artifactId>
+            <version>1.78.1</version>
```

Comment:
Versions must be defined in the main pom, not in the application module. We do it this way to keep consistency and avoid conflicts across the application